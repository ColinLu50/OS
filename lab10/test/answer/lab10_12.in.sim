7000
645
415
378
405
345
