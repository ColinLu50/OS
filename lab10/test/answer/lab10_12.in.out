FCFS
60 -> 308 -> 5 -> 351 -> 274 -> 262 -> 128 -> 168 -> 196 -> 224 -> 217 -> 248 -> 355 -> 239 -> 300 -> 41 -> 169 -> 5 -> 137 -> 89 -> 146 -> 190 -> 67 -> 35 -> 190 -> 233 -> 208 -> 169 -> 105 -> 99 -> 75 -> 226 -> 169 -> 329 -> 333 -> 256 -> 344 -> 21 -> 42 -> 352 -> 146 -> 271 -> 115 -> 282 -> 210 -> 263 -> 214 -> 95 -> 229 -> 19 -> 187 -> 99 -> 23 -> 65 -> 120 -> 240 -> 186 -> 310 -> 145 -> 176 -> 125 -> 129 -> 207 -> 22 -> 249 -> 288 -> 158
7000
SSTF
60 -> 65 -> 67 -> 75 -> 89 -> 95 -> 99 -> 99 -> 105 -> 115 -> 120 -> 125 -> 128 -> 129 -> 137 -> 145 -> 146 -> 146 -> 158 -> 168 -> 169 -> 169 -> 169 -> 176 -> 186 -> 187 -> 190 -> 190 -> 196 -> 207 -> 208 -> 210 -> 214 -> 217 -> 224 -> 226 -> 229 -> 233 -> 239 -> 240 -> 248 -> 249 -> 256 -> 262 -> 263 -> 271 -> 274 -> 282 -> 288 -> 300 -> 308 -> 310 -> 329 -> 333 -> 344 -> 351 -> 352 -> 355 -> 42 -> 41 -> 35 -> 23 -> 22 -> 21 -> 19 -> 5 -> 5
645
SCAN
60 -> 42 -> 41 -> 35 -> 23 -> 22 -> 21 -> 19 -> 5 -> 0 -> 65 -> 67 -> 75 -> 89 -> 95 -> 99 -> 105 -> 115 -> 120 -> 125 -> 128 -> 129 -> 137 -> 145 -> 146 -> 158 -> 168 -> 169 -> 176 -> 186 -> 187 -> 190 -> 196 -> 207 -> 208 -> 210 -> 214 -> 217 -> 224 -> 226 -> 229 -> 233 -> 239 -> 240 -> 248 -> 249 -> 256 -> 262 -> 263 -> 271 -> 274 -> 282 -> 288 -> 300 -> 308 -> 310 -> 329 -> 333 -> 344 -> 351 -> 352 -> 355
415
C-SCAN
60 -> 42 -> 41 -> 35 -> 23 -> 22 -> 21 -> 19 -> 5 -> 0 -> 383 -> 355 -> 352 -> 351 -> 344 -> 333 -> 329 -> 310 -> 308 -> 300 -> 288 -> 282 -> 274 -> 271 -> 263 -> 262 -> 256 -> 249 -> 248 -> 240 -> 239 -> 233 -> 229 -> 226 -> 224 -> 217 -> 214 -> 210 -> 208 -> 207 -> 196 -> 190 -> 187 -> 186 -> 176 -> 169 -> 168 -> 158 -> 146 -> 145 -> 137 -> 129 -> 128 -> 125 -> 120 -> 115 -> 105 -> 99 -> 95 -> 89 -> 75 -> 67 -> 65
378
LOOK
60 -> 42 -> 41 -> 35 -> 23 -> 22 -> 21 -> 19 -> 5 -> 65 -> 67 -> 75 -> 89 -> 95 -> 99 -> 105 -> 115 -> 120 -> 125 -> 128 -> 129 -> 137 -> 145 -> 146 -> 158 -> 168 -> 169 -> 176 -> 186 -> 187 -> 190 -> 196 -> 207 -> 208 -> 210 -> 214 -> 217 -> 224 -> 226 -> 229 -> 233 -> 239 -> 240 -> 248 -> 249 -> 256 -> 262 -> 263 -> 271 -> 274 -> 282 -> 288 -> 300 -> 308 -> 310 -> 329 -> 333 -> 344 -> 351 -> 352 -> 355
405
C-LOOK
60 -> 42 -> 41 -> 35 -> 23 -> 22 -> 21 -> 19 -> 5 -> 355 -> 352 -> 351 -> 344 -> 333 -> 329 -> 310 -> 308 -> 300 -> 288 -> 282 -> 274 -> 271 -> 263 -> 262 -> 256 -> 249 -> 248 -> 240 -> 239 -> 233 -> 229 -> 226 -> 224 -> 217 -> 214 -> 210 -> 208 -> 207 -> 196 -> 190 -> 187 -> 186 -> 176 -> 169 -> 168 -> 158 -> 146 -> 145 -> 137 -> 129 -> 128 -> 125 -> 120 -> 115 -> 105 -> 99 -> 95 -> 89 -> 75 -> 67 -> 65
345
